compiler/rustc_lint/src/noop_method_call.rs

```diff
@@ -129,6 +129,9 @@ impl<'tcx> LateLintPass<'tcx> for NoopMethodCall {
                 NoopMethodCallDiag { method: call.ident.name, receiver_ty, label: span },
             );
         } else {
+            if op == "borrow" {
```

Comment:
Why not just remove `sym::Borrow` from line 90 above? This seems weirdly hacky.

Reply:
borrow should still lint for no_op_method_call